### PR TITLE
Vue3: TimeSeriesTable

### PR DESCRIPTION
**Pull request by amarkensteijn**

### CODE REVIEW

File: src/lib/table/createTableData.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          value = event.y
+          p[j]++
+        }
+        result[s.id] = value
+      }
+    }
+    return result
+  })
+}
+
```
Comment (grijzea) [inline src/lib/table/createTableData.ts:50]: please add comment

File: src/lib/table/createTableData.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  if (chartSeriesArray === undefined) return []
+  const dateTimes = createDateTimes(chartSeriesArray, seriesRecord)
+
+  const chartSeries = uniqWith(
+    chartSeriesArray.filter((s) => seriesIds.includes(s.id)),
+    (a, b) => {
+      return a.id === b.id
+    },
+  )
+  const p = Array(seriesIds.length).fill(0)
```
Comment (grijzea) [inline src/lib/table/createTableData.ts:19]: provide a meaningfull name. p is bit confusing

File: src/components/table/TimeSeriesTable.vue [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  series: Record<string, Series>
+}
+
+const props = defineProps<Props>()
+
+const seriesIds = ref<string[]>([])
+const tableData = ref<Record<string, unknown>[]>([])
+const tableHeaders = ref<TableHeaders[]>([])
+
+onMounted(async () => {})
```
Comment (grijzea) [inline src/components/table/TimeSeriesTable.vue:59]: empty? please remove if not used

File: src/lib/table/createTableData.ts (hunk 2 above)
Comment (amarkensteijn) [reply src/lib/table/createTableData.ts:19]: Renamed to `pointers`

File: src/components/table/TimeSeriesTable.vue (hunk 3 above)
Comment (amarkensteijn) [reply src/components/table/TimeSeriesTable.vue:59]: Removed

**Pull request merged**